README.md

```diff
@@ -15,14 +19,12 @@ Detailed documentation can be found in our [GitHub Documenation Page](http://vmw
 
 ## Download
 
-**[Click here to download (Github releases)] (https://github.com/vmware/docker-volume-vsphere/releases)**
+**[Click here to download from Github releases](https://github.com/vmware/docker-volume-vsphere/releases)**
 
 The download consists of 2 parts:
 
-1. ESX: The ESX code is packaged as [a vib or an offline depot] (http://pubs.vmware.com/vsphere-60/index.jsp#com.vmware.vsphere.install.doc/GUID-29491174-238E-4708-A78F-8FE95156D6A3.html#GUID-29491174-238E-4708-A78F-8FE95156D6A3)
-2. VM Running Docker: The docker service is packaged as a deb or rpm file.
-   * Photon/RedHat => Download RPM
-   * Ubuntu => Download Deb.
+1. ESX: The ESX code is packaged as [a vib or an offline depot](http://pubs.vmware.com/vsphere-60/index.jsp#com.vmware.vsphere.install.doc/GUID-29491174-238E-4708-A78F-8FE95156D6A3.html#GUID-29491174-238E-4708-A78F-8FE95156D6A3)
+2. VM Running Docker: Plugin is available on [Docker store](https://store.docker.com/plugins/e15dc9d5-e20e-4fb8-8876-9615e6e6e852?tab=description).
 
 Pick the latest release and use the same version of ESX and VM release.
```

Comment:
Pick the latest release and use the same version of ESX and VM release. ->  Please use VIB and managed plugin corresponding to the same release.